internal/kafka/internal/workers/kafka_mgrs/accepted_kafkas_mgr.go

```diff
@@ -30,13 +30,14 @@ type AcceptedKafkaManager struct {
 }
 
 // NewAcceptedKafkaManager creates a new kafka manager
-func NewAcceptedKafkaManager(kafkaService services.KafkaService, quotaServiceFactory services.QuotaServiceFactory, clusterPlmtStrategy services.ClusterPlacementStrategy, bus signalbus.SignalBus, dataPlaneClusterConfig *config.DataplaneClusterConfig, clusterService services.ClusterService) *AcceptedKafkaManager {
+func NewAcceptedKafkaManager(kafkaService services.KafkaService, quotaServiceFactory services.QuotaServiceFactory, clusterPlmtStrategy services.ClusterPlacementStrategy, bus signalbus.SignalBus, dataPlaneClusterConfig *config.DataplaneClusterConfig, clusterService services.ClusterService, reconcilerConfig *workers.ReconcilerConfig) *AcceptedKafkaManager {
 	return &AcceptedKafkaManager{
 		BaseWorker: workers.BaseWorker{
 			Id:         uuid.New().String(),
 			WorkerType: "accepted_kafka",
 			Reconciler: workers.Reconciler{
-				SignalBus: bus,
+				SignalBus:        bus,
+				ReconcilerConfig: reconcilerConfig,
 			},
```

Comment:
I am still a bit conflicted on the idea of each worker creating the `Reconciler` object. 

I am leaning more toward injecting the reconciler and assigning it, just like
```go
func NewAcceptedKafkaManager(kafkaService services.KafkaService, quotaServiceFactory services.QuotaServiceFactory, clusterPlmtStrategy services.ClusterPlacementStrategy, dataPlaneClusterConfig *config.DataplaneClusterConfig, clusterService services.ClusterService, reconciler workers.Reconciler) *AcceptedKafkaManager {
	return &AcceptedKafkaManager{
		BaseWorker: workers.BaseWorker{
			Id:         uuid.New().String(),
			WorkerType: "accepted_kafka",
			Reconciler: reconciler,
		},
		kafkaService:           kafkaService,
		quotaServiceFactory:    quotaServiceFactory,
		clusterPlmtStrategy:    clusterPlmtStrategy,
		dataPlaneClusterConfig: dataPlaneClusterConfig,
		clusterService:         clusterService,
	}
}
```
The pattern I am suggesting above is already done for the cluster_mgr: https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/f8e2a70b344f460abde5b36e1e4345a6e4f4f8fb/internal/kafka/internal/workers/clusters_mgr.go#L109
 
And apply it throughout. What do you think? @VanillaSpoon

Reply:
Yea, I think thats the right move